### PR TITLE
Hotswap colors for instances

### PR DESCRIPTION
**Pull request by CedricGuillemet**
WIP

### CODE REVIEW

File: src/Meshes/instancedMesh.ts [hunk 1]
```diff
@@ -558,7 +558,11 @@ Mesh.prototype.registerInstancedBuffer = function (kind: string, stride: number)
     this._userInstancedBuffersStorage.strides[kind] = stride;
     this._userInstancedBuffersStorage.sizes[kind] = stride * 32; // Initial size
     this._userInstancedBuffersStorage.data[kind] = new Float32Array(this._userInstancedBuffersStorage.sizes[kind]);
-    this._userInstancedBuffersStorage.vertexBuffers[kind] = new VertexBuffer(this.getEngine(), this._userInstancedBuffersStorage.data[kind], kind, true, false, stride, true);
+    let kindSubstitution = kind;
+    if (kind === VertexBuffer.ColorKind) {
+        kindSubstitution = VertexBuffer.ColorInstanceKind;
```
[the remark this answers is not in the document]
Comment (CedricGuillemet) [reply src/Meshes/instancedMesh.ts:563]: Yes.

**Review by deltakosh — APPROVED** [review]

**Comment by deltakosh** [comment]
I think we can merge no?

**Comment by CedricGuillemet** [comment]
Nop. Still a draft.

**Comment by CedricGuillemet** [comment]
I'll see if I find some time before release. Closing for now.

**Pull request closed without merging**